src/extension.ts

```diff
@@ -89,7 +88,7 @@ export async function activate(context: vscode.ExtensionContext) {
 
 	const jobManager = new JobManager(
 		persistedState?.jobs.map((job) => mapPersistedJobToJob(job)) ?? [],
-		persistedState?.acceptedJobsHashes as JobHash[],
+		persistedState?.appliedJobsHashes as JobHash[],
```

Comment:
what's the type of `persistedState?.appliedJobsHashes`?

Reply:
did not noticed that it can be undefined. fixed that.